startup/gui/project.py

```diff
@@ -48,15 +48,22 @@
 
 def __scriptAdded( container, script ) :
 
+
 	variables = script["variables"]
 	if "projectName" not in variables :
-		projectName = variables.addMember( "project:name", IECore.StringData( "default" ), "projectName" )
-		projectName["name"].setFlags( Gaffer.Plug.Flags.ReadOnly, True )
+		variables.addChild(Gaffer.CompoundDataPlug.MemberPlug("projectName", flags=(Gaffer.Plug.Flags.Default ) & ~Gaffer.Plug.Flags.Serialisable))
```

Comment:
Making these plugs non-dynamic breaks scripts in standalone gaffer - resulting in the following error when loading them :

```
ERROR : Line 11 of /Users/john/gaffer/projects/default/scripts/testVariables.gfr 
KeyError: "'projectName' is not a child of 'variables'"
```